src/rpc/server.cpp

```diff
@@ -252,7 +252,7 @@ UniValue stop(const UniValue& params, bool fHelp)
 /**
  * Call Table
  */
-static const CRPCCommand vRPCCommands[] =
+extern const CRPCCommand vRPCCommands[] =
```

Comment:
I don't like exposing this array using `extern`, it is an implementation detail. It also doesn't interoperate with commands that are added later on by modules (in this case the wallet in `walletRegisterRPCCommands`). Now you need to pull from everywhere...

My suggestion would be to add a `ListCommands` to `TableRPC`, that returns a `std::vector<std::string>` of commands, and use that.